static/app/views/monitors/monitorForm.tsx

```diff
@@ -208,24 +209,30 @@ class MonitorForm extends Component<Props> {
                   case 'interval':
                     return (
                       <Fragment>
-                        <NumberField
-                          name="config.schedule.frequency"
-                          label={t('Frequency')}
-                          placeholder="e.g. 1"
-                          help={t(
-                            'The amount of intervals that pass between executions of the cron job.'
-                          )}
-                          required
-                        />
-                        <SelectField
-                          name="config.schedule.interval"
-                          label={t('Interval')}
-                          options={INTERVALS}
-                          help={t(
-                            'The interval on which the frequency will be applied. 1 time every X amount of (minutes, hours, days)'
-                          )}
-                          required
-                        />
+                        <FieldGroup>
+                          <Field
+                            label={t('Frequency')}
+                            help={t(
+                              'The amount of time between each job execution. Example, every 5 hours.'
+                            )}
+                            stacked
+                            required
+                          />
+                          <StyledNumberField
+                            name="config.schedule.frequency"
+                            label={t('Frequency')}
+                            placeholder="e.g. 1"
+                            hideLabel
+                            required
+                          />
+                          <StyledSelectField
+                            name="config.schedule.interval"
+                            label={t('Interval')}
+                            options={INTERVALS}
+                            hideLabel
+                            required
+                          />
```

Comment:
Something feels like it might be a little wacky here wrt how the accessibility of these works

Reply:
yea thats fair.. sorry kind of getting this out last minute to unblock crons launch 😅 . Is there a less hacky way to stack form fields like this?

Reply:
Let's just do this but figure out something later. I can't think of a quick solution off the top of myhead

Reply:
tyty, sounds good